src/main/java/calculator/StringCalculatorRunner.java

```diff
@@ -0,0 +1,27 @@
+package calculator;
+
+import java.util.Scanner;
+import java.util.regex.Pattern;
+
+public class StringCalculatorRunner {
+    private Scanner scanner;
```

Comment:
```suggestion
    private final Scanner scanner = new Scanner(System.in);
```
생성자에서 매번 scanner 객체를 생성할 필요는 없어보이네요

Reply:
변경하였습니다 :)